tasks/nginx_setup.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Install system prerequisites
+  apt:
```

Comment:
In my opinion we should use 'package' instead of 'apt'

Reply:
Are you sure that the same package names work?

Reply:
Yes, because 'package' is only a wrapper around. For example: On ubuntu ansible will use apt in the background

Reply:
If the platform under 'meta/main.yml' is ubuntu only it is ok for me to leave it as 'apt' for the moment